kernel/base/src/main/java/com/twosigma/beaker/jvm/object/HelpLink.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
In my opinion this solution is more readable creating hashmap for each text and url, furthermore i think this solution allows us to reuse HelpLink in any other Kernels if we would.

What you think?